projects/organization-management/src/app/components/user-budget-form/user-budget-form.component.spec.ts

```diff
@@ -58,14 +49,15 @@ describe('User Budget Form Component', () => {
   it('should write current currency into the form after init', () => {
     fixture.detectChanges();
 
-    expect(component.form.get('currency').value).toBe('USD');
+    expect(component.model.currency).toBe('USD');
   });
 
-  it('should display form input fields after creation', () => {
+  it('should have form input fields after creation', () => {
     fixture.detectChanges();
 
-    expect(element.querySelector('[controlname=orderSpentLimit]')).toBeTruthy();
-    expect(element.querySelector('[formControlname=budget]')).toBeTruthy();
-    expect(element.querySelector('[formControlname=budgetPeriod]')).toBeTruthy();
+    expect(JSON.stringify(component.fields)).toContain('currency');
```

Comment:
I think using JSON.stringify could lead to problems when the fields contain circular values (which they often do). If the test doesn't mock FormlyForm anymore, we could also add a more meaningful test here.